go-controller/cmd/ovnkube/ovnkube.go

```diff
@@ -228,14 +241,14 @@ func runOvnKube(ctx *cli.Context) error {
 			}
 			clusterController.ClusterServicesSubnet = servicesSubnet.String()
 		}
-		clusterController.NodePortEnable = nodePortEnable
+		runForever = true
```

Comment:
I think you meant to initialize this with 'false'

Reply:
I think it should be "true". When we running as "master", we need to runForever to allocate subnets. When we run as "node", we need to runForever to get CNI pod events.

Reply:
Sorry, I meant the initial value. Not the assignment at this line. 
Anyway, it's good because I realise that golang's nil value for boolean is 'false'.

Reply:
Yeah. Initializing it generally causes a 'make lint' failure.